python-sdk/src/langwatch/telemetry/span.py

```
@@ -289,6 +296,26 @@ def get_span_context(self) -> Optional[SpanContext]:
 
+    def add_link(self, attributes: Optional[Dict[str, Any]] = None) -> None:
+        """Add a link to this span."""
+        if self._span is None:
+            warn("Cannot add link - no span available")
+            return
+        try:
+            self._span.add_link(attributes)
+        except Exception as e:
+            warn(f"Failed to add link to span: {str(e)}")
+
+    def set_status(self, status: Status, description: Optional[str] = None) -> None:
```

Ruff (0.8.2)

Redefinition of unused `set_status` from line 254

(F811)